pom.xml

```diff
@@ -47,7 +47,7 @@
 
         <deps.apache-sshd.version>0.12.0</deps.apache-sshd.version>
         <deps.jsch.version>0.1.52</deps.jsch.version>
-        <deps.ssh-credentials-plugin.version>1.13-SNAPSHOT</deps.ssh-credentials-plugin.version>
+        <deps.ssh-credentials-plugin.version>2.0-SNAPSHOT</deps.ssh-credentials-plugin.version>
```

Comment:
Actually looking at the whole PR, you are actually introducing the anti-pattern in this PR, so my personal view is :-1: on introducing the anti-pattern... note that this is just an advisory view from a Maven PMC member and committer and is not binding on the plugin maintainer